GridKit/AutomaticDifferentiation/Enzyme/SparseWrapper.hpp

```diff
@@ -325,15 +328,15 @@ namespace GridKit
          * @param[in] wb - Bus variables
          * @param[in,out] jac - Jacobian
          */
-        static void eval(ModelT*                                            model,
-                         size_t                                             n_res,
-                         size_t                                             n_var,
-                         const std::map<IdxT, IdxT>&                        res_indices,
-                         const std::map<IdxT, IdxT>&                        var_indices,
-                         ScalarT*                                           y,
-                         ScalarT*                                           yp,
-                         ScalarT*                                           wb,
-                         GridKit::LinearAlgebra::COO_Matrix<ScalarT, IdxT>& jac)
+        static void eval(ModelT*                                              model,
+                         size_t                                               n_res,
+                         size_t                                               n_var,
+                         const std::map<IdxT, IdxT>&                          res_indices,
+                         const std::map<IdxT, IdxT>&                          var_indices,
+                         ScalarT*                                             y,
+                         ScalarT*                                             yp,
+                         ScalarT*                                             wb,
+                         GridKit::LinearAlgebra::COO_Matrix<real_type, IdxT>& jac)
         {
```

Comment:
Is ti possible to replace 
```suggestion
                         MatrixT<real_type, IdxT>& jac)
{
```
where we define `MatrixT` witha `using` statement?

Reply:
Similar at a few other places in this file.

Reply:
Addressed in [244630c](https://github.com/ORNL/GridKit/pull/305/commits/244630c2f3899029955fc7881e2bf98784f1b758)